src/pandas_profiling/report/structure/variables/render_categorical.py

```diff
@@ -403,11 +403,13 @@ def render_categorical(config: Settings, summary: dict) -> dict:
 
     overview_items = []
 
-    if length:
+    # length isn't being computed for categorical in spark
+    if length and "max_length" in summary:
```

Comment:
will `max_length` be present for pandas?

Reply:
Yes, working for pandas.